### ISSUE TITLE
access denied when start process

### ISSUE THREAD

**Issue by ZijianHan** [josh4364/IL2cppStartProcess#6]
Hi, 
I'm trying to use this lib wrapper to start an external process (out of the Unity project folder) but I got the access denied error. Any possible solution for it? It would not be so convenient in my case to put the executable inside of Unity project.
Br
Zijian

**Comment by josh4364** [comment]
Not sure, are you sure you have access to the exe file your referencing?

**Comment by ZijianHan** [comment]
Hi, yes and I also tried to launch Unity as admin, still no access issue. But anyway, thanks for sharing the lib, we'll look deeper into issue later.

**Comment by josh4364** [comment]
Yeah, sorry I can't help as much, I don't have much time outside of work :(
If there ends up being a change to this file needed, let me know and I can do it.

**Comment by AllFatherGray** [comment]
> Hi, yes and I also tried to launch Unity as admin, still no access issue. But anyway, thanks for sharing the lib, we'll look deeper into issue later.

@ZijianHan Did you find a workaround?
